azure-pipelines.yml

```diff
@@ -1,13 +1,15 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
+name: $(Build.SourceBranchName)_$(Build.Reason)_$(devops_buildNumber)
 
 pool:
-  vmImage: 'Ubuntu 16.04'
+  vmImage: 'vs2017-win2016'
+
+variables:
+  devops_buildNumber: $[counter(format(''), 1300)]
+  APPVEYOR_REPO_BRANCH: $(Build.SourceBranchName)
+  APPVEYOR_REPO_COMMIT: $(Build.SourceVersion)
```

Comment:
Eventually we can remove these mentions of appveyor, i'm assuming?

Reply:
yes, i did this so that app veyor keeps working in case this doesn't. Just a fallback mechanism